modules/nw-sriov-dpdk-example-intel.adoc

```diff
@@ -101,7 +101,7 @@ spec:
     image: <DPDK_image> <2>
     securityContext:
      capabilities:
```

Comment:
Please add "runAsUser: 0" under the securityContext

securityContext structure should look as follows: 
securityContext:
  runAsUser: 0
  capabilities:
    add: ["IPC_LOCK","SYS_RESOURCE","NET_RAW"]